cmd/actions.go

```diff
@@ -36,6 +37,27 @@ var (
 			},
 		},
 	}
+
+	subcmdActionsSetRunnerToken = &cli.Command{
+		Name:    "set-runner-token",
+		Usage:   "Set a new token for a runner to as register token",
```

Comment:
But isn't "Set a new token" simply "adding a token"?

Reply:
Because once the new token is added, all old tokens will be invalided. Only one token will be valid. So that I use `set` rather than `add`.